data/brands/amenity/payment_terminal.json

```diff
@@ -618,6 +618,17 @@
         "payment:cash:USD": "yes",
         "payment:coins": "no"
       }
+    },
+    {
+      "displayName": "ფეიბოქსი",
+      "locationSet": {"include": ["ge"]},
+      "tags": {
+        "amenity": "payment_terminal",
+        "brand": "ფეიბოქსი",
```

Comment:
Please add a brand:wikidata tag, creating a new Wikidata item if necessary. For information on editing Wikidata, please see https://github.com/osmlab/name-suggestion-index/wiki/Editing-Wikidata

Reply:
Would be good to do the same for
https://github.com/osmlab/name-suggestion-index/blob/0bfac56dbdad1aeae8bf1ce857e99869f734c8d4/data/brands/amenity/payment_terminal.json#L583
Oppa operates PayBox, Teso, Oppa Service, Asterbit.

Reply:
> Please add a brand:wikidata tag, creating a new Wikidata item if necessary. For information on editing Wikidata, please see https://github.com/osmlab/name-suggestion-index/wiki/Editing-Wikidata

Added Wikidata and operator